src/main/java/bio/terra/service/filedata/azure/util/BlobContainerClientFactory.java

```diff
@@ -1,90 +1,71 @@
 package bio.terra.service.filedata.azure.util;
 
+import static bio.terra.common.ValidationUtils.requiresNotBlank;
```

Comment:
In general, we don't use `static` imports, except in tests, and there usually only for matchers and asserts.

Reply:
Fixed.